packages/gasket-plugin-manifest/lib/express.js

```diff
@@ -0,0 +1,18 @@
+const baseConfig = require('./base-config');
+/**
+ * If configured, serve the resolved manifest.json
+ * @param  {Gasket} gasket The gasket API
+ * @param  {Express} app gasket's express server
+ * @async
+ */
+async function express(gasket, app) {
+  const { config } = gasket;
+  const { staticOutput } = (config && config.manifest || {});
+  const { path } = (config && config.manifest || {});
+
+  app.get(staticOutput || path || baseConfig.path, (req, res) => {
```

Comment:
`staticOutput` is where the file lives on disk (if outputted), and not the URL path.
```suggestion
  app.get(path || baseConfig.path, (req, res) => {
```